README.md

```diff
@@ -101,14 +101,14 @@ This SIG follows the [standard operating model](https://github.com/cncf/toc/blob
 - TOC Liaison: [Matt Klein](https://twitter.com/mattklein123)
 - Chairs: [Lee Calcote](https://twitter.com/lcalcote), [Ken Owens](https://twitter.com/kenowens12)
 
-In accordance with the [elections and terms](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections) follow the [CNCF SIG definition](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections).
+In accordance with the [elections and terms](https://github.com/cncf/toc/blob/master/tags/cncf-sigs.md#elections) follow the [CNCF TAG definition](https://github.com/cncf/toc/blob/master/tags/cncf-sigs.md#elections).
 
 ## Communications
 
-- Slack Channel ([#sig-network](https://app.slack.com/client/T08PSQ7BQ/CMG237Z5Z))
-- Join [SIG-Network](mailto:sig-network@lists.cncf.io) mailer at [lists.cncf.io](https://lists.cncf.io)
-- Repo: [https://github.com/cncf/sig-network](https://github.com/cncf/sig-network)
+- Slack Channel ([#tag-network](https://app.slack.com/client/T08PSQ7BQ/CMG237Z5Z))
+- Join [TAG-Network](mailto:tag-network@lists.cncf.io) mailer at [lists.cncf.io](https://lists.cncf.io)
+- Repo: [https://github.com/cncf/tag-network](https://github.com/cncf/tag-network)
 - Meetings: 
-  - Time: Network SIG meets the 1st and 3rd Thursday of every month at 11am Pacific (here's [a link](https://goo.gl/eyutah) to a public Google calendar that you can subscribe to).
+  - Time: Network TAG meets the 1st and 3rd Thursday of every month at 11am Pacific (here's [a link](https://goo.gl/eyutah) to a public Google calendar that you can subscribe to).
   - Location: Zoom - [https://zoom.us/my/cncfsignetwork](https://zoom.us/my/cncfsignetwork)
```

Comment:
Yep, Zooms have changed

Reply:
@amye, is this correct: https://zoom.us/my/cncftagnetwork? It seems to be.